src/ops/ops.go

```diff
@@ -105,8 +107,12 @@ func (e *ExecCommandError) DetailedError() string {
 	return fmt.Sprintf("failed executing %s %v, env vars %v, error %s, waitStatus %d, Output \"%s\"", e.Command, e.Args, e.Env, e.ExitErr, e.WaitStatus, e.Output)
 }
 
-// ExecCommand executes command.
 func (o *ops) ExecCommand(liveLogger io.Writer, command string, args ...string) (string, error) {
+	return o.execCommand(liveLogger, "", command, args...)
```

Comment:
What is the reason for that? Why not adding `cmd.Dir=workdir` directly in `ExecCommand` ?

Reply:
I did not want to change all the command calling.most of them will transfer "" as workDir which means current process directory